### PR TITLE
Update dependencies

### PR DESCRIPTION
**Pull request by albertborsos**
Update `sentry/sentry` to `~1.8` and `raven-js` to `~3.17`.

Resolve https://github.com/hellowearemito/yii2-sentry/issues/17

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -30,9 +30,12 @@
             "mito\\sentry\\tests\\": "tests/"
         }
     },
-    "extra": {
-        "asset-installer-paths": {
-            "bower-asset-library": "vendor/bower"
+    "config": {
+        "fxp-asset": {
```
Comment (thiagotalma) [inline composer.json:35]: Hello @albertborsos!
It is worth remembering that the Yii2 officially abandoned the fxp-asset.
https://github.com/yiisoft/yii2-app-advanced/blob/master/composer.json#L35

Comment (thiagotalma) [reply composer.json:35]: https://github.com/yiisoft/yii2/commit/d3f97f7e64f9ced72a418b70d4e2ad9f3f23d902
https://github.com/yiisoft/yii2-app-advanced/commit/4a892a65c122bcea294fd93fc4fb860b2789dd58

Comment (albertborsos) [reply composer.json:35]: Thanks for the information, I will update it!

Comment (nkovacs) [reply composer.json:35]: Only the root package's config is used, so this is only used when working on this package.

**Comment by Alex-Bond** [comment]
Will you merge it anytime soon?

**Review by nkovacs — APPROVED** [review]
LGTM

**Comment by nkovacs** [comment]
Codesniffer is failing though, you need to specify a version (probably 2.x). The coding standards package allows 3.x and it probably broke it.

**Comment by albertborsos** [comment]
Thank you for your patience. I released it under version `1.0.3`

**Pull request merged**